src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandler.java

```diff
@@ -38,4 +51,85 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    private CallbackResponse validateValues(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        List<String> errors = new ArrayList<>();
+
+        if (caseData.getDocumentUploadWitness1() != null) {
+            List<Element<UploadEvidenceWitness>> dateList = caseData.getDocumentUploadWitness1();
+            dateList.forEach(date -> {
+                if (date.getValue().getWitnessOption1UploadDate().isAfter(LocalDate.now())) {
+                    errors.add("Invalid date: date entered must not be in the future.");
+                }
+            });
+        }
+
+        if (caseData.getDocumentUploadWitness3() != null) {
```

Comment:
Why is this UploadWitness 3, rather than 2?

Reply:
There's 4 documentUploadWitness1/2/3/4 versions only 1 and 3 have a date field associated that needs validated